test/helpers/queue.ts

```diff
@@ -1,20 +1,74 @@
 import { stub, SinonStub } from 'sinon';
 import * as qf from '../../app/util/queue/queue-factory';
+import * as util from '../../app/backends/workflow-orchestration/util';
 import { MemoryQueue } from './memory-queue';
+import { Logger } from 'winston';
+import logger from '../../app/util/log';
+import { getWorkFromDatabase } from '../../app/backends/workflow-orchestration/work-item-polling';
+import { WorkItemQueueType } from '../../app/util/queue/queue';
 
 let serviceQueues;
+let typeQueues;
+
+/**
+ * Process the scheduler queue. This function is only used for tests since they won't have a
```

Comment:
Maybe add a little more detail on what "Process the scheduler queue" entails?

Reply:
Done.